docs/rules/prefer-arrow-callback.md

```diff
@@ -37,6 +37,27 @@ foo(function() { return this.a; });
 foo(function bar(n) { return n && n + bar(n - 1); });
 ```
 
+## Options
+
+`prefer-arrow-callback` receives a single, optional argument which is an options object.
+
+### functionsWithThis
+
```

Comment:
I think we need to work on this option name, I'm not sure it's obvious what this means.